Sources/Controllers/2.0/OAQuickActionListViewController.m

```diff
@@ -135,7 +145,10 @@ -(CGFloat) getToolBarHeight
 
 - (IBAction)backPressed:(id)sender
 {
-    [self.navigationController popViewControllerAnimated:YES];
+    if (_isShouldBeClosedByBackButton)
```

Comment:
see ```- (IBAction) backButtonClicked:(id)sender``` in OASuperViewController. This functionality is already present and should be used instead by adding the outlet to the correct method